packages/vue/src/stories/releases/v0.13.x/v0.13.7/v0.13.6.stories.mdx

```diff
@@ -0,0 +1,10 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Releases/v0.13.x - Latest/v0.13.7 - Latest/Change log" />
+
+# v0.13.7
+
+## 🐛 Fixes
+
+## 🧹 Chores:
+- chore: improved structure for releases section
```

Comment:
@ymaheshwari1 Thanks for cleaning up this mess :) I'd like to merge it, but could you please change the name of this file to v0.13.7.stories.mdx? Thanks for your contribution.

Reply:
Done

Reply:
Thank you very much @ymaheshwari1